docs/content/getting-started/create-store.mdx

```diff
@@ -97,4 +97,14 @@ class MyProgram {
 
 </TabItem>
 
-</Tabs>
\ No newline at end of file
+<TabItem value={SupportedLanguage.CURL} label={languageLabelMap.get(SupportedLanguage.CURL)}>
+
+```console
+curl -XPOST $FGA_API_SCHEME://$FGA_API_HOST/stores \
```

Comment:
```suggestion
curl -X POST $FGA_API_HOST/stores \
```
To be consistent with other curl command

Reply:
Thanks!